Automattic/wp-calypso
# Add/stats to jetpack complete lightbox

## Proposed Changes

* Update Jetpack Complete lightbox to include Jetpack Stats
* Update the comparison page to have Stats as a product, not just a feature

<img width="1097" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/3197f13f-d7ac-4373-a47c-7fd70a6dea33">
<img width="1222" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/659da730-c734-462b-a2bf-4576ec913de1">


## Testing Instructions



* Do one of the following:
  * Open the Jetpack Cloud live link
  * `git checkout add/stats-to-jetpack-complete-lightbox` and `yarn start-jetpack-cloud` locally
* Visit `/pricing#jetpack_complete` and confirm that you see a full item for Stats in the "Products included" list 
* Confirm that Stats is also in the sidebar list, with the correct price ($10 if your currency is USD)
* Visit `/features/comparison`, confirm that Stats under the "Growth" section links to https://jetpack.com/stats, and that it matches the screenshot above

## Pre-merge Checklist
